composer.json

```diff
@@ -6,23 +6,22 @@
     "keywords": ["keboola", "snowflake", "db", "extractor"],
     "authors": [
         {
-            "name": "Erik Žigo",
-            "email": "erik@keboola.com"
+            "name": "Keboola",
+            "email": "devel@keboola.com"
         }
     ],
     "require": {
-        "php": ">=5.6",
-        "keboola/db-extractor-common": "~5.2",
-        "keboola/php-temp": "^0.1.6",
-        "keboola/php-csv-db-import": "~2.3.0",
-        "squizlabs/php_codesniffer": "^2.6",
-        "vkartaviy/retry": "^0.2"
+        "php": ">=7.1",
+        "keboola/db-extractor-common": "~9.1",
+        "keboola/php-temp": "^1.0",
+        "keboola/php-csv-db-import": "~2.3.0"
     },
     "require-dev": {
-        "phpunit/phpunit": "4.6.*",
+        "phpunit/phpunit": "^7.0",
         "codeclimate/php-test-reporter": "^0.3",
+        "keboola/coding-standard": "^6.0",
         "keboola/storage-api-client": "^4.14",
-        "keboola/php-csv-db-import": "~2.3.0"
+        "phpstan/phpstan": "^0.8"
```

Comment:
0.10.3 is latest afaik

Reply:
as you are adding a new dependency you should set it to latest available IMHO... even though it will install correctly latest version even with this version constraint